### DISCUSSION TITLE
Is there a way to put a message / declaimer / note on the Pipeline page?

### DISCUSSION THREAD

**Discussion by mukteswarp** [concourse/concourse#7594] [Help & Support]
Hi,

I am thinking about putting a note / disclaimer on the pipeline page, so that any user before running the pipeline can refer the same before executing. Also is it possible to post on page about the health or last commit of the source repo along with the last build version on the pipeline page for concourse.

Thanking you in advance.

**Comment by taylorsilva** [comment]
No such feature exists. There have been similar requests: https://github.com/concourse/concourse/issues/569 https://github.com/concourse/concourse/issues/2025 and possibly others I didn't find.

**Reply by mukteswarp** [reply]
Thanks @taylorsilva

**Comment by grignards** [comment]
Hi.
I (and my company) would also be interested to have a description for pipelines. Judging by the discussion on #2025 others are too.
If a contribution adding this feature is welcomed I'd be interested to work on it.

**Reply by taylorsilva** [reply]
We always welcome contributions :) It's not on the main teams roadmap right now so community help is probably the only way that feature will get done.

**Reply by grignards** [reply]
OK, if this is fine I'll start by implementing descriptions for pipelines/jobs and resources based on @mockersf ideas in https://github.com/concourse/concourse/issues/569#issuecomment-483070125 and https://github.com/concourse/concourse/issues/569#issuecomment-492827325.

**Reply by taylorsilva** [reply]
Yup, that works! Feel free to comment in that issue with any questions/blockers you have while implementing, we (or at least I) will see it :)
